tools/spectral/.spectral.yaml

```diff
@@ -155,7 +155,36 @@ rules:
         function: pattern
         functionOptions:
           match: "^(dev|qa|stage|prod)(,(dev|qa|stage|prod))*$"
-
+  
+  x-xgen-soa-migration-extension:
```

Comment:
do you also plan to re-introduce the custom method rule as part of this PR?

Reply:
not in this one, will do in another one